modApiServer/src/org/aion/api/server/pb/ApiAion0.java

```diff
@@ -908,7 +908,7 @@ public ApiAion0(IAionChain ac) {
             try {
                 Message.rsp_syncInfo rsp = Message.rsp_syncInfo.newBuilder().setChainBestBlock(sync.chainBestBlkNumber)
                         .setNetworkBestBlock(sync.networkBestBlkNumber).setSyncing(!sync.done)
-                        .setMaxImportBlocks(sync.blocksRequestMax).build();
+                        .setMaxImportBlocks(0).build();
```

Comment:
Earlier I saw that we requested in batches of 24, could we make that a constant and set it here as well?